src/OSPSuite.Core/Workspace.cs

```diff
@@ -48,7 +49,7 @@ public void UpdateJournalPathRelativeTo(string projectFileFullPath)
       protected void ReleaseMemory()
       {
          //This might free some C++ objects still  being held into memory
-         this.GCCollectAndCompact();
```

Comment:
Having it as extension method was stupid. This is better as a simple static method